azext_edge/tests/edge/orchestration/test_upgrade_unit.py

```diff
@@ -178,7 +177,7 @@ def _generate_trains(**trains) -> dict:
 
 
 @pytest.mark.parametrize("no_progress", [False, True])
-@pytest.mark.parametrize("require_instance_update", [False, True])
+# @pytest.mark.parametrize("require_instance_update", [False, True])
```

Comment:
A few commented out lines here - should these be removed?

Reply:
crap